pkg/controller/node/controller_utils.go

```diff
@@ -320,3 +331,14 @@ func terminatePods(kubeClient clientset.Interface, recorder record.EventRecorder
 	}
 	return complete, nextAttempt, nil
 }
+
+func HealthyQPSFunc(nodeNum int, defaultQPS float32) float32 {
+	return defaultQPS
+}
+
+func ReducedQPSFunc(nodeNum int, defaultQPS float32) float32 {
+	if nodeNum > 20 {
```

Comment:
We reduce if there are more than 20 nodes and stop evicting when there's less. Added a comment.